larq_compute_engine/tflite/kernels/bconv2d_impl.h

```diff
@@ -167,9 +168,12 @@ inline void BConv2D(
     // The RHS tensor has this shape which we bitpack along the last dimension
     //  [batch, output_height, output_width, k * bitwidth]
     RuntimeShape packed_input_shape;
-    ce::core::packbits_tensor<ce::core::BitpackOrder::Optimized>(
-        result_shape, result_data, params.input_offset, packed_input_shape,
-        packed_input_data);
+    {
+      ruy::profiler::ScopeLabel lable("Bitpack activations (after im2col)");
+      ce::core::packbits_tensor<ce::core::BitpackOrder::Optimized>(
+          result_shape, result_data, params.input_offset, packed_input_shape,
+          packed_input_data);
+    }
```

Comment:
Why do we need the brackets here, is this for the profiler to work correctly?

Reply:
Yes. These 'labels' start when they are constructed and finish when they are deconstructed. They are deconstructed when their scope ends, and their scope is now determined by the brackets.
Sidenote: due to your comment I spotted the typo `lable` -> `label`, thanks!